get.sh

```diff
@@ -626,40 +628,55 @@ fi
 checkRepoSHA()
 {
 	output_file="$TESTDIR/TKG/SHA.txt"
-	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file"
-	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file
+	if [ -e ${output_file} ]; then
+		echo "rm $output_file"
+		rm ${output_file}
+	fi
+
+	sha_file="$TESTDIR/TKG/SHA.txt"
+	testenv_file="$TESTDIR/testenv/testenv.properties"
+	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $sha_file"
+	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $sha_file
+
+	echo "$TESTDIR/getTestenvProperties.sh --repo_dir $1 --output_file $testenv_file --repo_name $2"
+	$TESTDIR/getTestenvProperties.sh --repo_dir $1 --output_file $testenv_file --repo_name $2
+
+
 }
 
 checkTestRepoSHAs()
 {
 	echo "check adoptium repo and TKG repo SHA"
 
-	output_file="$TESTDIR/TKG/SHA.txt"
-	if [ -e ${output_file} ]; then
-		echo "rm $output_file"
-		rm ${output_file}
-	fi
-
-	checkRepoSHA "$TESTDIR"
-	checkRepoSHA "$TESTDIR/TKG"
+	checkRepoSHA "$TESTDIR" "ADOPTIUM"
+	checkRepoSHA "$TESTDIR/TKG" "TKG"
 }
 
 checkOpenJ9RepoSHA()
 {
 	echo "check OpenJ9 Repo sha"
-	checkRepoSHA "$TESTDIR/openj9"
+	checkRepoSHA "$TESTDIR/openj9" "OPENJ9"
 }
 
+checkOpenJDKBinSha()
+{
+	echo "check openjdkbinary Repo sha"
+	checkRepoSHA "$SDKDIR/openjdkbinary" "JDK${JDK_VERSION}"
+}
 
 parseCommandLineArgs "$@"
 if [[ "$USE_TESTENV_PROPERTIES" == true  ]]; then
 	source ./testenv/testenv.properties
 fi
 
+echo "Clearing testenv.properties"
+> ./testenv/testenv.properties
```

Comment:
I do not think we need the echo

Reply:
Why do we need line 657?

Reply:
Line 657 clears testenv.properties, so that we will be writing to an empty file. Without this line, we would be appending to the existing testenv.properties file.